lib/elixir/lib/enum.ex

```diff
@@ -3473,6 +3475,40 @@ defmodule Enum do
     reduce(enumerable, 0, &+/2)
   end
 
+  @doc """
+  Maps and sums the given enumerable in one pass.
+
+  Raises `ArithmeticError` if `fun` returns a non-numeric value.
+
+  ## Examples
+
+      iex> Enum.sum_by([%{count: 1}, %{count: 2}, %{count: 3}], fn x -> x.count end)
+      6
+
+      iex> Enum.sum_by(1..3, fn x -> x ** 2 end)
+      14
+
+      iex> Enum.sum_by([], fn x -> x.count end)
+      0
+
+  Filtering can be achieved by returning `0` to remove elements:
```

Comment:
```suggestion
  Filtering can be achieved by returning `0` to ignore elements:
```

I think that current wording may be a little bit misleading. I prefer such wording. Though I think that this whole paragraph could be removed as it "should be obvious".

Reply:
I like `ignore`, thank you for the suggestion.
I agree it should be obvious, yet I think it's worth it if it can help people think to use this style vs the much less efficient:

```elixir
[1, -2, 3] |> Enum.filter(fn x -> x > 0 end) |> Enum.sum()
```